### PR TITLE
Fix javadoc warning.

### PR DESCRIPTION
**Pull request by zhangkun83**

### CODE REVIEW

File: benchmarks/src/main/java/io/grpc/benchmarks/ByteBufOutputMarshaller.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 import java.io.InputStream;
 
 /**
- * Simple {@link MethodDescriptor#Marshaller} for Netty's {@link ByteBuf}.
+ * Simple {@link io.grpc.MethodDescriptor.Marshaller} for Netty's {@link ByteBuf}.
```
Comment (carl-mastrangelo) [inline benchmarks/src/main/java/io/grpc/benchmarks/ByteBufOutputMarshaller.java]: This still nags in my IDE.  Maybe just make a {@code} ref to it?

Comment (zhangkun83) [inline benchmarks/src/main/java/io/grpc/benchmarks/ByteBufOutputMarshaller.java]: Did your IDE snag before the change? What does it say?

Comment (carl-mastrangelo) [inline benchmarks/src/main/java/io/grpc/benchmarks/ByteBufOutputMarshaller.java]: Just yellow squiggles.  I'm not sure the intersection of IDEs that people use are smart enough to resolve this ref.  Not that this change make it any worse.

**Comment by carl-mastrangelo** [comment]
LGTM

**Pull request merged**